doc/radosgw/lua-scripting.rst

```diff
@@ -71,6 +71,13 @@ To remove a package from the allowlist:
   # radosgw-admin script-package rm --package={package name}
 
 
+To remove a specific version of a package from the allowlist:
```

Comment:
please explain what happens when a different version of the package (or the package without a version) already exists